bin/v2.js

```diff
@@ -207,22 +248,17 @@ const getDeployedVersionList = (bucketName, bucketPath) => {
 const formatAndLimitDeployedVersions = (versions, size) => {
   const returnedVersions = [];
 
-  _.reverse(
-    _.sortBy(
-      versions,
-      ['lastModified']
-    )
-  )
+  _.reverse(_.sortBy(versions, ['lastModified']))
     .slice(0, size).forEach(version => {
 
     const {base, ext} = path.parse(version.filepath);
     const split = base.split(".");
 
-    // This check is technically not necessary assuming getDeployedVersionList only returns versions
-    // matching this criteria.
+    // This check is strictly not necessary assuming getDeployedVersionList only returns versions
+    //  matching these criteria.
```

Comment:
Since we're doing comment suggestions, I'll add my own
```suggestion
    // matching these criteria.
```